src/functionalTest/java/uk/gov/hmcts/reform/pip/account/management/controllers/SystemAdminAccountTest.java

```diff
@@ -173,12 +132,11 @@ void testCreateSystemAdminAccountRequestExceeded() throws Exception {
     }
 
     @Test
-    @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = "classpath:add-admin-users.sql")
-    void testGetAzureUserInfo() throws Exception {
+    @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = SQL_ADD_ADMIN)
+    void testUserCanCreateSystemAdminAccountBadRequest() throws Exception {
```

Comment:
Rename this to testCreateSystemAdminReturnsBadRequest()

Reply:
Renamed the test:)